### PR TITLE
[INLONG-9879][Sort] Fix the inlongmsg-rowdata-kv format parse head error

### LINKED ISSUE #9879: [Bug][Sort] Sort inlongmsg-rowdata-kv format parse head has error
### What happened

企业微信截图_7db97085-6fcd-4764-9a08-309fd519f8d5

### What you expected to happen

will parse error while head attr contain correct args such as:
streamId=XXX

### How to reproduce

producer inlongmsg-kv msg and use sort inlongmsg-rowdata-kv format parse it.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by baomingyu**
- Fixes #9879

### CODE REVIEW

**Review by dockerzhang — APPROVED** [review]

File: inlong-sort/sort-formats/format-row/format-inlongmsg-binlog/src/main/java/org/apache/inlong/sort/formats/inlongmsgbinlog/InLongMsgBinlogUtils.java [hunk 1]
```diff
@@ -117,10 +118,14 @@ public static InLongMsgHead parseHead(String attr) {
...
         } else if (attributes.containsKey(INLONGMSG_ATTR_TID)) {
             streamId = attributes.get(INLONGMSG_ATTR_TID);
+        } else if (attributes.containsKey(INLONGMSG_ATTR_INTERFACE_NAME)) {
+            streamId = attributes.get(INLONGMSG_ATTR_INTERFACE_NAME);
         } else {
             throw new IllegalArgumentException(
                     "Could not find " + INLONGMSG_ATTR_STREAM_ID
-                            + " or " + INLONGMSG_ATTR_TID + " in attributes!");
+                            + " or " + INLONGMSG_ATTR_TID
+                            + " or " + INLONGMSG_ATTR_INTERFACE_NAME
```
Comment (aloyszhang) [inline inlong-sort/sort-formats/format-row/format-inlongmsg-binlog/src/main/java/org/apache/inlong/sort/formats/inlongmsgbinlog/InLongMsgBinlogUtils.java:127]:
```suggestion
                            + " and " + INLONGMSG_ATTR_TID
                            + " and " + INLONGMSG_ATTR_INTERFACE_NAME
```

File: inlong-sort/sort-formats/format-row/format-inlongmsg-csv/src/main/java/org/apache/inlong/sort/formats/inlongmsgcsv/InLongMsgCsvUtils.java [hunk 2]
```diff
@@ -65,10 +66,13 @@ public static InLongMsgHead parseHead(String attr) {
             streamId = attributes.get(INLONGMSG_ATTR_STREAM_ID);
         } else if (attributes.containsKey(INLONGMSG_ATTR_TID)) {
             streamId = attributes.get(INLONGMSG_ATTR_TID);
+        } else if (attributes.containsKey(INLONGMSG_ATTR_INTERFACE_NAME)) {
+            streamId = attributes.get(INLONGMSG_ATTR_INTERFACE_NAME);
         } else {
             throw new IllegalArgumentException(
                     "Could not find " + INLONGMSG_ATTR_STREAM_ID
                             + " or " + INLONGMSG_ATTR_TID
+                            + " or " + INLONGMSG_ATTR_INTERFACE_NAME
```
Comment (aloyszhang) [inline inlong-sort/sort-formats/format-row/format-inlongmsg-csv/src/main/java/org/apache/inlong/sort/formats/inlongmsgcsv/InLongMsgCsvUtils.java:75]:
```suggestion
                            + " and " + INLONGMSG_ATTR_TID
                            + " and " + INLONGMSG_ATTR_INTERFACE_NAME
```

File: inlong-sort/sort-formats/format-rowdata/format-inlongmsg-rowdata-binlog/src/main/java/org/apache/inlong/sort/formats/inlongmsgbinlog/InLongMsgBinlogUtils.java [hunk 3]
```diff
@@ -73,17 +74,20 @@ public static InLongMsgHead parseHead(String attr) {
...
-            streamId = attributes.get(INLONGMSG_ATTR_INTERFACE_TID);
         } else {
             throw new IllegalArgumentException(
-                    "Could not find " + INLONGMSG_ATTR_INTERFACE_NAME +
-                            " or " + INLONGMSG_ATTR_INTERFACE_ID +
-                            " or " + INLONGMSG_ATTR_INTERFACE_TID + " in attributes!");
+                    "Could not find " + INLONGMSG_ATTR_STREAM_ID
+                            + " or " + INLONGMSG_ATTR_INTERFACE_TID
+                            + " or " + INLONGMSG_ATTR_INTERFACE_NAME
+                            + " or " + INLONGMSG_ATTR_INTERFACE_ID + " in attributes!");
```
Comment (aloyszhang) [inline inlong-sort/sort-formats/format-rowdata/format-inlongmsg-rowdata-binlog/src/main/java/org/apache/inlong/sort/formats/inlongmsgbinlog/InLongMsgBinlogUtils.java:90]:
```suggestion
                            + " and " + INLONGMSG_ATTR_INTERFACE_TID
                            + " and " + INLONGMSG_ATTR_INTERFACE_NAME
                            + " and " + INLONGMSG_ATTR_INTERFACE_ID + " in attributes!");
```

File: inlong-sort/sort-formats/format-rowdata/format-inlongmsg-rowdata-kv/src/main/java/org/apache/inlong/sort/formats/inlongmsgkv/InLongMsgKvUtils.java [hunk 4]
```diff
@@ -53,18 +53,17 @@ public static InLongMsgHead parseHead(String attr) {
...
+        } else if (attributes.containsKey(INLONGMSG_ATTR_INTERFACE_NAME)) {
+            streamId = attributes.get(INLONGMSG_ATTR_INTERFACE_NAME);
         } else {
             throw new IllegalArgumentException(
-                    "Could not find " + INLONGMSG_ATTR_STREAMID
-                            + " or " + INLONGMSG_ATTR_INTERFACE_NAME
-                            + " or " + INLONGMSG_ATTR_INTERFACE_TID + " in attributes!");
+                    "Could not find " + INLONGMSG_ATTR_STREAM_ID
+                            + " or " + INLONGMSG_ATTR_INTERFACE_TID
+                            + " or " + INLONGMSG_ATTR_INTERFACE_NAME + " in attributes!");
```
Comment (aloyszhang) [inline inlong-sort/sort-formats/format-rowdata/format-inlongmsg-rowdata-kv/src/main/java/org/apache/inlong/sort/formats/inlongmsgkv/InLongMsgKvUtils.java:66]:
```suggestion
                            + " and " + INLONGMSG_ATTR_INTERFACE_TID
                            + " and " + INLONGMSG_ATTR_INTERFACE_NAME + " in attributes!");
```

File: inlong-sort/sort-formats/format-rowdata/format-inlongmsg-rowdata-csv/src/main/java/org/apache/inlong/sort/formats/inlongmsgcsv/InLongMsgCsvUtils.java [hunk 5]
```diff
@@ -65,20 +65,20 @@ public static InLongMsgHead parseHead(String attr) {
...
-            streamId = attributes.get(INLONGMSG_ATTR_STREAMID);
         } else {
             throw new IllegalArgumentException(
-                    "Could not find " + INLONGMSG_ATTR_INTERFACE_NAME +
-                            " or " + INLONGMSG_ATTR_INTERFACE_ID +
+                    "Could not find " + INLONGMSG_ATTR_STREAM_ID +
                             " or " + INLONGMSG_ATTR_INTERFACE_TID +
-                            " or " + INLONGMSG_ATTR_STREAMID + " in attributes!");
+                            " or " + INLONGMSG_ATTR_INTERFACE_NAME +
+                            " or " + INLONGMSG_ATTR_INTERFACE_ID + " in attributes!");
```
Comment (aloyszhang) [inline inlong-sort/sort-formats/format-rowdata/format-inlongmsg-rowdata-csv/src/main/java/org/apache/inlong/sort/formats/inlongmsgcsv/InLongMsgCsvUtils.java:81]:
```suggestion
                            " and " + INLONGMSG_ATTR_INTERFACE_NAME +
                            " and " + INLONGMSG_ATTR_INTERFACE_ID + " in attributes!");
```

Comment (baomingyu) [reply inlong-sort/sort-formats/format-rowdata/format-inlongmsg-rowdata-csv/src/main/java/org/apache/inlong/sort/formats/inlongmsgcsv/InLongMsgCsvUtils.java:81]: it need anyone here ,not all of them

**Review by aloyszhang — APPROVED** [review]

**Pull request merged**